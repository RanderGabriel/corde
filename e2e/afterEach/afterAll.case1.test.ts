import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("should print on console", async () => {
  const command = Utils.buildCommandWithConfigPath("afterAll", "bot_case1.test.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(mockProcess).toBeCalledWith(0);
  expect(stdout).toMatchSnapshot();
});
