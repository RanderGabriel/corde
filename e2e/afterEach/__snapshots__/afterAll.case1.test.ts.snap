// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should print on console 1`] = `""`;
